<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_battle</class>
 <widget class="QDialog" name="w_battle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>530</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="rounds">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>第1轮 炮击战</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>| |</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout_0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QProgressBar" name="progressBar01">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::TopToBottom</enum>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label03">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label02">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label05">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label01">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label06">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label04">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QProgressBar" name="progressBar05">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QProgressBar" name="progressBar03">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QProgressBar" name="progressBar06">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="progressBar02">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QProgressBar" name="progressBar04">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>550</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_1">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label13">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="progressBar12">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QProgressBar" name="progressBar11">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QProgressBar" name="progressBar16">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label11">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label16">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label12">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QProgressBar" name="progressBar14">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QProgressBar" name="progressBar13">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label15">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QProgressBar" name="progressBar15">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label14">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
